commands/create_cluster.go

```diff
@@ -416,8 +416,13 @@ func definitionFromFlags(def clusterDefinition, args addClusterArguments) cluste
 	}
 
 	if def.Scaling.Min == 0 && def.Scaling.Max == 0 {
-		def.Scaling.Min = int64(len(def.Workers))
-		def.Scaling.Max = int64(len(def.Workers))
+		def.Scaling.Min = int64(args.numWorkers)
+		def.Scaling.Max = int64(args.numWorkers)
+	}
+
+	if def.Scaling.Min == 0 && def.Scaling.Max == 0 && args.numWorkers == 0 {
```

Comment:
We need to take care of this case here, we need to default these settings because we need to send atleast 1 worker for e.g. `cpu` and so on. The API will only create 1 worker otherwise (which is correct behaviour).

Reply:
if the default correct behavior is creating 1 worker why we use a different one?

Reply:
The case is the following:
- We start an old cluster without autoscaling and submit the worker array with 1 worker (to have cpu).
- API receives a request without scaling values but one worker in workers array
- API can't decide if an old gsctl wants to create a cluster with one worker or if it should default to three workers

-> solution: we send the scalign values defaulted here :)